ui/static/ui/js/listing_resources.jsx

```diff
@@ -430,6 +432,7 @@ define('listing_resources', ['react', 'jquery', 'lodash', 'utils'],
     var ListingPage = React.createClass({
       render: function() {
         return <div>
```

Comment:
One thought (not necessary to implement with this PR, from my pt of view). Would it make sense to always add in an id or a class to divs within react components (and possibly matching close div comments), so we can hook into them more easily from CSS, and identify their functions when viewed in the browser? If we have a lot of vanilla ```<div>```s sprinkled around, it could get harder over time to keep things straight. Maybe react gives us what we need? React ids tend to be inscrutable to the naked eye. Thoughts?